### ISSUE TITLE
60hz still on

### ISSUE THREAD

**Issue by berryforcoffee** [gurrgur/er-patcher#74]
Using 12.2

Everything except the 60hz works. Uncapped frame-rate works, no vignette, no chromatic abberation etc. (I did not test for animation distance, rune loss, or ultra-wide)

My game does start, I saw some people saying there's won't. I'm on Windows 11.

This is my command line: python er-patcher --rate 90 --all -- %command%

**Comment by polsrepo** [comment]
Yeah, it's a known issue. I'm currently trying to fix it on my end with no luck so far.

**Comment by polsrepo** [comment]
BTW, running the game in Borderless Fullscreen removes the 60hz lock but prevents you from using HDR

**Comment by berryforcoffee** [comment]
> BTW, running the game in Borderless Fullscreen removes the 60hz lock but prevents you from using HDR

Yeah, I'm aware thx. HDR is a bigger deal than uncapped FPS to me. Spent a lot of dough for my tv lol. Game looks effin stunning.

**Comment by polly125** [comment]
> > BTW, running the game in Borderless Fullscreen removes the 60hz lock but prevents you from using HDR
> 
> Yeah, I'm aware thx. HDR is a bigger deal than uncapped FPS to me. Spent a lot of dough for my tv lol. Game looks effin stunning.

https://www.nexusmods.com/eldenring/mods/4513?tab=description

click on Override Fullscreen state and you can now turn on hdr

**Comment by AmbiguousInteractive** [comment]
> > > BTW, running the game in Borderless Fullscreen removes the 60hz lock but prevents you from using HDR
> > 
> > 
> > Yeah, I'm aware thx. HDR is a bigger deal than uncapped FPS to me. Spent a lot of dough for my tv lol. Game looks effin stunning.
> 
> https://www.nexusmods.com/eldenring/mods/4513?tab=description
> 
> click on Override Fullscreen state and you can now turn on hdr

Where is that located?

**Comment by berryforcoffee** [comment]
> > > > BTW, running the game in Borderless Fullscreen removes the 60hz lock but prevents you from using HDR
> > > 
> > > 
> > > Yeah, I'm aware thx. HDR is a bigger deal than uncapped FPS to me. Spent a lot of dough for my tv lol. Game looks effin stunning.
> > 
> > 
> > https://www.nexusmods.com/eldenring/mods/4513?tab=description
> > click on Override Fullscreen state and you can now turn on hdr
> 
> Where is that located?

Once the mod is up and running open the interface in-game, click on view -> advanced settings -> Override FullScreen State After which you must restart the game.

**Comment by AmbiguousInteractive** [comment]
> > > > > BTW, running the game in Borderless Fullscreen removes the 60hz lock but prevents you from using HDR
> > > > 
> > > > 
> > > > Yeah, I'm aware thx. HDR is a bigger deal than uncapped FPS to me. Spent a lot of dough for my tv lol. Game looks effin stunning.
> > > 
> > > 
> > > https://www.nexusmods.com/eldenring/mods/4513?tab=description
> > > click on Override Fullscreen state and you can now turn on hdr
> > 
> > 
> > Where is that located?
> 
> Once the mod is up and running open the interface in-game, click on view -> advanced settings -> Override FullScreen State After which you must restart the game.

Awesome, works great!
